automation/check-patch.unit-test.sh

```diff
@@ -8,15 +8,15 @@
 # automation/check-patch.e2e-k8s.sh
 
 teardown() {
-    cp $(find . -name "*junit*.xml") $ARTIFACTS
+    cp unit-test-junit.xml api/unit-test-junit.xml $ARTIFACTS
```

Comment:
the --output-dir can be used instead this copy ?

Reply:
Ahh right!

Reply:
Ahh no if we do so they will be overwitten the root and api.

Reply:
can't each have other file name ?
unless i am missing something, not sure about the flow here

Reply:
This is what I did

Reply:
Spyglass expect files like the following
```yaml
- lens:                                                                     
        name: junit                                                             
      required_files:                                                           
        - artifacts/junit.*\.xml         
```

Reply:
so just rename it right ?
but i think that even if you will have the right file, it will fail parsing it because the XSD is changed